torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1520,6 +1520,11 @@ void ProcessConstantValueMap(Node* n, int opset_version) {
   // For outputs, only update static shapes. For input, we update symbolic
   // shapes also. ONNX If can have different types on different branches, skip
   // here.
+
+  // Update reliable before processing constantvaluemap prevents unreliable
+  // nodes making static shape
```

Comment:
```suggestion
  // Update the shape reliability for each node before processing ConstantValueMap to prevent unreliable
  // nodes from producing static shapes
```
nit: wording